ui/src/components/search/ResultItem.tsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
-import ReactHtmlParser from "react-html-parser";
-import { MathJax } from "better-react-mathjax";
+import MathJax from "react-mathjax2";
 
 import { ArticleIdentifier, Result } from "@/types";
 import PublicationInfo from "../shared/PublicationInfo";
 import Authors from "../shared/Authors";
```

Comment:
if you are using alias while importing functions from utils and types, would be good to use them also while importing components as well